azure/ConsiderationsForServiceDesign.md

```diff
@@ -90,6 +97,13 @@ Before releasing your API plan to invest significant design effort, get customer
 
 :ballot_box_with_check: **YOU SHOULD**  capture what you have learned during the preview stage and share these findings with your team and with the API Stewardship Board.
 
+## Communicate Deprecations
```

Comment:
@JeffreyRichter @markweitzel Should we introduce guidance around scream testing so that teams are allowed to start returning different status code after the deadline has passed before effectively pulling the API version support?

Reply:
If the service is still running with the latest version, then the client's request will reach the endpoint, the service will examine the passed-in api-version query parameter, determine that the version is no longer valid and return a 400? with an Azure error response code of "Invalid api-version"?

Mark, we should standardize the status code and Azure error code/message when a client makes a request to an invalid api-version and make service team return this consistently.